uktrade/data-hub-api
# CPS-628 Fix Django Admin link to investment project record on FE

### Description of change

The **VIEW ON SITE** button on the **Change investment project** page in Django Admin should take the user to the project page on the frontend of Data Hub. Instead of displaying the project, users are currently seeing a **Page not found** error message. This is because '/details' is missing from the end of the url link.
This PR corrects the issue.

### To test

- In Django Admin scroll down to **INVESTMENT** and select **Investment Projects**
- Select a project
- Click on the **VIEW ON SITE** button in the top right
<img width="982" alt="Screenshot 2024-11-15 at 15 15 52" src="https://github.com/user-attachments/assets/39383cb7-ac71-4c2d-953d-206bf8168397">

- The relevant investment project page on Data Hub should be displayed without errors

### Checklist


  <details>
  <summary>Explanation</summary>
  
  The branch should not be stale or have conflicts at the time reviews are requested.
  
  </details>


### General points

<details>
<summary><strong>Other things to check</strong></summary><p></p>

* Make sure `fixtures/test_data.yaml` is maintained when updating models
* Consider the admin site when making changes to models
* Use select-/prefetch-related field lists in views and search apps, and update them when fields are added
* Make sure the README is updated e.g. when adding new environment variables

</details>

See [docs/CONTRIBUTING.md](https://github.com/uktrade/data-hub-api/blob/main/docs/CONTRIBUTING.md) for more guidelines.
